app/i18n/en/namespaces/feed.js

```diff
@@ -5,6 +5,7 @@ const feed = {
     comment: ' {{action}} {{object}}',
     deleteTopic: ' {{action}} {{object}}',
     createTopic: ' {{action}} {{object}}',
+    action: ' {{action}} {{object}}',
```

Comment:
I don't really understand why you did it this way? Doesn't this make it impossible for future translations to change the order of `{{ action }}` and `{{ object }}`? Or am I misunderstanding something? Why is there a `deleteTopic`, `createTopic` *and* a generic `action`?